docs/Using-Fleet/MDM-macOS-setup.md

```diff
@@ -0,0 +1,75 @@
+# macOS setup
+
+In Fleet you can customize the first time macOS setup experience for your end users.
+
+You can require end users to authenticate with your identity provider (IdP) and agree to and end user license agreement (EULA) before they can use their new Mac.
+
+You can customize the macOS Setup Assistant and choose to show or hide specific panes.
+
+Also, you can bootstrap new Macs with your configuration management tool of choice (ex. Munki, Chef, or Puppet).
+
+## End user authentication
+
+> This feature is currently in development.
```

Comment:
What's the point of this and the other similar one below?

Reply:
Features are currently being implemented. I want to give engineers working on the UI a valid URL to link to.